home/templates/home/docs/applicant_guide.html

```diff
@@ -661,6 +661,8 @@ <h2 id='final-application'><a href='#final-application'>Final application</a></h
 		Carefully review the project description to familiarize yourself with the provided tasks and milestones. Once you have a clear understanding of the project, create a timeline outlining the tasks you plan to work on during the internship. Some interns have found it helpful to break down their project tasks by weeks.
 		Be prepared to adjust your timeline as needed to accommodate unexpected developments or changes in project scope. Make sure to take into account any time commitments you have during the Outreachy internship round.
 		If you are still working on your contributions and need more time, you can leave this blank and edit your application later.
+        In addition to your project timeline, plan to incorporate milestones where you share drafts of your work with your mentors and the community.
```

Comment:
We may want to tell applicants to adopt the PDCA (plan-do-check-adjust) method (https://en.wikipedia.org/wiki/PDCA) when thinking about their project execution. We could maybe expand the concept of a draft to exemplify what we mean in different contexts (i.e. for a coding project, mentioning work being developed on a specific branch, or discussions on issues and/or mailing lists; for a documentation project, it may be an assessment of the current state of the docs). 🤔

Reply:
If we're going to be expanding this to apply to different contexts, the project timeline is going to need its own dedicated section in the applicant's guide so we can add as much info as needed.

I also think we need a different issue for this, since this is a bit out of scope with the issue #572. Maybe this issue/PR should focus on just making the applicants less dependent on mentors and a different PR for a more detailed guide on writing project timelines. What do you think?

Reply:
Also, once we have a detailed guide for project timelines, we can reference it here with a link.

Reply:
Agreed!